client/my-sites/plugins/index.js

```diff
@@ -102,6 +103,17 @@ export default function () {
 			clientRender
 		);
 
+		if ( config.isEnabled( 'marketplace-yoast' ) ) {
+			page(
+				'/plugins/domain/:site?',
```

Comment:
Could you elaborate on the rationale behind this route structure? Maybe I'm just unfamiliar with the project, but it doesn't make much sense to me as it is, especially considering the existing route structure in `/plugins`.

Reply:
Hey @tyxla this is meant to be the domains step inside the plugins section. @olaolusoga @JanaMW27 is there any other route structure you would find preferable to be shown on the domains step? I just added this with the view that this is a domains page shown inside the plugin context.

CC - @dzver @southp

Reply:
Could you show me some more context? Design, or a conversation? It's quite possible I'm missing the context and that's the reason why this piece feels odd to me to live in the plugins section.